src/commercial/krux.js

```
@@ -0,0 +1,105 @@
+const Superstore = require('superstore');
+const store = new Superstore('session', 'next-krux');
+const oAds = require('o-ads');
+
+const addPixel = (src) => {
+	const img = new Image();
+	img.src = src;
+};
+
+const addScript = (src) => {
+	const s = document.createElement('script');
+	s.type = 'text/javascript';
+	s.async = true;
+	s.src = src;
+	document.body.appendChild(s);
+};
+
+const frequencyCap = (name, limit, fn) => {
+	const key = `scriptExecCount_${name}`;
+	store.get(key).then(val => {
+		val = val || 0;
+		if(parseInt(val) < limit) {
+			fn();
+			store.set(key, ++val);
+		}
```

i assume superstore fails silently, so we end up *not* running any scripts on browsers that don't do localStorage (eg, full up, iOS private mode etc.)